### DISCUSSION TITLE
Visualization for ToothGrowth

### DISCUSSION THREAD

**Discussion by Scientist328455** [royal-statistical-society/datavisguide#31] [General]
I think more information can be plotted in the ToothGrowth example with less ink. Particularly the bimodal nature of the data in the OJ 0.5 mg/day dose group. In my opinion most of the bar in a bar chart is wasted ink. They can almost entirely be replaced with box plots or scatter plots denoting the mean.

![image](https://github.com/royal-statistical-society/datavisguide/assets/125690836/cfbd7a27-a877-4c71-ab56-8480d03b5e0a)

```R
library(ggplot2)

ToothGrowth$dose <- factor(ToothGrowth$dose)
ToothGrowth.summary <- aggregate(. ~ dose + supp, mean, data=ToothGrowth)

position_jitterdodge(
  jitter.width = NULL,
  jitter.height = 0,
  dodge.width = 0.5,
  seed = NA
)
ggplot(ToothGrowth, aes(x=dose, y=len, color=supp)) +
  geom_point(position = position_jitterdodge()) + 
  stat_summary(geom='errorbar', aes(ymax = ..y.., ymin = ..y..),
                position=position_dodge(width = 0.75), width=0.5) +
  scale_color_brewer(palette='Set1') +
  ylab('Tooth Length') +
  xlab('Dose (mg/day)') + 
  ggtitle('Tooth Growth') +
  theme_bw()
```

**Comment by gimickwu** [comment]
I really like that showing individual data points gives a sens of the variability!
